.github/workflows/tests.yml

```diff
@@ -43,7 +43,7 @@ jobs:
       fail-fast: true
       matrix:
         lumen: [5.5.*, 5.6.*, 5.7.*, 5.8.*, 6.*, 7.*]
```

Comment:
We should probably bump Lumen too. We'll get it fixed up for next time.

Reply:
I don't believe Lumen has been tagged yet